### ISSUE TITLE
NC 14 Announces don't appear in Activity and Notifications

### ISSUE THREAD

**Issue by Gabuzomeuh** [nextcloud/announcementcenter#116]
Hi.

When posting in "Announcement Center", announces don't appear in "Activity" and don't trigger "Notifications".

Checkboxes in "Advanced options" are both checked.
Announce is really published.
No errors in NC logs.

When enabled, "Comments" do work as usual.

Announcement Center used to work perfectly under NC 13.0.6, this problem only appears since upgrade to 14.0.1.

### Server configuration

**Nextcloud version:** 14.0.1

**Upgraded from:** 13.0.6

**Comment by nickvergessen** [comment]
Is your cron job executed?
You can see this in `index.php/settings/admin`

**Comment by Gabuzomeuh** [comment]
Hi.

Yes cronjob was executed without errors, even in apache logs.

I won't be able to help you further since i gave up NC14 and reverted to NC13.

I encountered too many issues, misbehavior and losses of performance, even on a fresh install. Having struggled for the last 2 days trying to figure out and fix most of them, i finally decided to give up.

I do think all of you, devs, are working really hard on this release and i respect your work, but i can't consider NC 14.0.1 as stable. Maybe i will again in a few milestones.

Thank you for your work, you already made a great job.

**Comment by criwe** [comment]
Same here.
Best

**Comment by nickvergessen** [comment]
I just tested this again and it works when the cron job is executed.
If this does not work for you, please check the number of jobs in the `oc_jobs` table of your database. If it is higher than 20, you need to change your cron jobs to system cron.

**Comment by FloMiLe** [comment]
Hello @nickvergessen!

I have got two questions:

1. I also experience the issue that a new announcement does neither trigger a notification nor an activity for neither the poster nor a sender.
1. Following your suggestion, I checked whether there is an issue with cron. But I don't understand what this means: I have got 26 entries in `oc_jobs` but in `index.php/settings/admin` it says that the cron job (system cron) was run successfully. Are those 26 a problem in general? And what would I need to do to fix it?

Thanks in advance for your reply!
